packages/lab/src/Wizard/WizardTitle/WizardTitle.d.ts

```diff
@@ -0,0 +1,55 @@
+import * as React from "react";
+import { StandardProps } from "@material-ui/core";
+import { Breakpoint } from "@material-ui/core/styles/createBreakpoints";
+
+export type HvWizardTitleClassKey = "root";
+
+export type HvWizardTitleProps = StandardProps<
+  React.HTMLAttributes<HTMLDivElement>,
+  HvWizardTitleClassKey
+> & {
+  /**
+   * Title for the wizard.
+   */
+  title: string;
+  /**
+   * Shows the summary button.
+   */
+  hasSummary: boolean;
+  /**
+   * An object containing all the labels for the wizard header.
+   */
+  labels: {
+    /**
+     * Summary button label.
+     */
+    summary: string;
+  };
+  /**
+   * Current tab to check if it's last page or first to disable previous button and swap between next and submit button.
+   */
+  tab: number;
+  /**
+   * Function to change the tab when pressing previous and next buttons.
+   */
+  changeTab: any;
+  /**
+   * Custom object to define type, size and width of the StepNavigation component
+   */
+  customStep?: {
```

Comment:
If you put Pick<HvStepNavigationProps, "type" | "size" | "width"> it will be always protected for future changes on StepNavigation component